docs/v3-UPGRADE-GUIDE.md

```diff
@@ -33,7 +33,16 @@ Prior to v3.x, we included a `browser` field in the package.json file. Since nod
 
 ## Dropped the `res.textConverted()` function
 
-If you want charset encoding detection, please use [fetch-charset-detection] package.
+If you want charset encoding detection, please use the [fetch-charset-detection] package ([documentation][fetch-charset-detection-docs]).
+
+```js
+const fetch = require("node-fetch");
+const {convertBody} = require("fetch-charset-detection");
```

Comment:
should it just be `const convertBody = require("fetch-charset-detection");` ?

we only expose one function.

Reply:
The original problem was that using `export default` for ECMAScript would break when using CommonJS so I changed it to a named export to work consistently since using `module.export` would also indirectly break Typedoc.

I've now figured out that I can just export them using both methods so Typedoc, ECMAScript and CommonJS all work.